client/index.html

```diff
@@ -1,10 +1,23 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="en" ng-app="TwitterWallApp">
     <head>
         <meta charset="utf-8">
         <title>TWITTER WALL</title>
+        
+        <!-- ANGULAR -->
+        <script src="https://ajax.googleapis.com/ajax/libs/angularjs/1.5.5/angular.min.js"></script>
+        <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.12.4/jquery.min.js"></script>
```

Comment:
Don't really need to include JQuery yet, but will be needed if we use angular material so fine to leave in